examples/reinforcement_learning/actor_critic_cartpole.py

```diff
@@ -0,0 +1,119 @@
+"""
+Title: Actor Critic Method
+Author: [Apoorv Nandan](https://twitter.com/NandanApoorv)
+Date created: 2020/05/13
+Last modified: 2020/05/13
+Description: Implement Actor Critic Method in CartPole environment
+"""
+"""
+## Setup
+"""
+
+import gym
+import numpy as np
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow import math
+from tensorflow.keras import losses
+from tensorflow.keras import layers
+from tensorflow.keras import optimizers
```

Comment:
You don't need these imports, just use `tf.` and `keras.` for thins like `tf.math`, `keras.losses`, etc.

Reply:
Got it.

I was actually wondering about the exact best practice in such cases. 
If only a single optimizer and loss function are being instantiated, we use `keras.losses.Huber` or `keras.optimizers.Adam` and if multiple things from a submodule are being used (like multiple layers), we import `from tf.keras import layers` and use `layers.Dense` etc. Is this understanding correct?